pkg/types/strint.go

```diff
@@ -41,3 +41,10 @@ func (s *StrInt64) UnmarshalJSON(body []byte) error {
 
 	return nil
 }
+
+func (s *StrInt64) String() string {
```

Comment:
you can use value receiver here?

Reply:
No, I can't because we have a JSON marshaler on a pointer receiver. The combination of a pointer and value is not recommended by Golang